DataConnectors/Symantec Endpoint Protection/Connector_Syslog_SymantecEndpointProtection.json

```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "SymantecEndpointProtection",
+    "title": "Symantec Endpoint Protection", 
+    "publisher": "Broadcom", 
+    "descriptionMarkdown": "The [Broadcom Symantec Endpoint Protection (SEP)](https://www.broadcom.com/products/cyber-security/endpoint/end-user/enterprise) connector allows you to easily connect your SEP logs with Azure Sentinel This gives you more insight into your organization's network and improves your security operation capabilities.", 
```

Comment:
Add '.' (period) after Azure Sentinel and before 'This gives you more insight'.

Reply:
corrected and committed.